cartridge/failover.lua

```diff
@@ -192,6 +200,49 @@ local function apply_config(mod)
     )
 end
 
+local function reconfigure_all()
+    local confapplier = require('cartridge.confapplier')
+    local all_roles = require('cartridge.roles').get_all_roles()
+
+    -- WARNING: implicit yield
+    -- The event may arrive while two-phase commit is in progress.
+    -- We should wait for the appropriate state.
+    local state = confapplier.wish_state('RolesConfigured', math.huge)
```

Comment:
Why `math.huge`? Not some configurable value? e.g. two-phase commit timeout.

Reply:
Because the only reason to abort reconfiguration is fiber cancellation and inappropriate state.
Bot of them are handled in this expression: wish_state returns immediately if state transition is impossible (e.g. OperationError -> RolesConfigured)